definitions/tools/homer_tag_directory.cwl

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'Creating the homer tag directory'
+baseCommand: ["/bin/bash", "homer_tag_directory.sh"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "mgibio/homer:0.1"
+    - class: ResourceRequirement
+      ramMin: 32000
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'homer_tag_directory.sh'
+        entry: |
+                set -o pipefail
+                set -o errexit
+
+                name="homer_tag_directory"
+                bam=$1
+                reference_fasta=$2
+
+                export PATH=$PATH:/opt/homer/bin
+                if [[ ! -d $name/homer ]];then
+                    mkdir -p $name/homer
+                fi
```

Comment:
there's no point in seeing if the dir exists, since we know it doesn't exist up front in the CWL output dir - you can always just create it!

Reply:
Changed it